README.md

```diff
@@ -489,6 +489,31 @@ A final optional field, called `indent`, exists to define the indentation method
 for that language. Topiary defaults to two spaces `"  "` if it cannot find the
 indent field in any configuration file for a specific language.
 
+#### Specifying the grammar
+Topiary can fetch and build the grammar for you, or a grammar can be provided by
+some other method. To have Topiary fetch the grammar for you, specify the
+`grammar.source.git` attribute of a language:
+```nickel
+nickel = {
+  extensions = ["ncl"],
+  grammar.source.git = {
+    git = "https://github.com/nickel-lang/tree-sitter-nickel",
+    rev = "43433d8477b24cd13acaac20a66deda49b7e2547",
+  },
+},
+```
+
+To specify a prebuilt grammar, specify the `grammar.source.path` attribute:
+```nickel
+nickel = {
+  extensions = ["ncl"],
+  grammar.source.path = "/home/erin/Projects/tweag/tree-sitter-nickel/result/parser",
+},
+```
```

Comment:
What does a "prebuilt grammar" look like; presumably an `.so` file? It might be useful to elaborate on this here, to make it clearer for readers.